README.md

```diff
@@ -233,7 +231,7 @@ Lightweight [Swiper](https://swiperjs.com/get-started) wrapper. NostoSwiper comp
 In order to use Swiper modules the names must be passed as *strings*. This is due to the handling of the modules on demand.
```

Comment:
What do we mean by "passed as strings"? 😕

Reply:
Not directly related but since we are updating the docs, just raising it here

Reply:
should we use names instead?

Reply:
Maybe, we can try to elaborate it more. The explanation seems bit vague